packages/landing/src/components/daterange.tsx

```diff
@@ -46,7 +28,9 @@ export class DateRange extends Component {
         this.setState({ dateBefore: `${event.target.value}-12-31T23:59:59.999Z` });
         break;
     }
-    this.scheduleUpdateConfig();
+    Config.map.dateRange.dateAfter = this.state.dateAfter;
```

Comment:
would be good to check if these have actually changed.

Reply:
`handleChange` is the callback attached to the `onChange` attribute of the two `<input>` elements, so will only ever be called when a value has changed.

Reply:
Further to this, changes made by @Wentao-Kuang to add `Config.map.setFilterDateRange` (which updates the date range values on the config and emits `filter` and `change` events if they have changed) mean that we don't need to emit anything here, and the events will only be emitted if needed.